data/arweave/assets/uZwgZTwGhoYj_KuDbeqFx3Kv7G-FQHlJ5YA6i8Omu7s/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "CrypNote Token Test",
+  "symbol": "Note",
+  "web": "https://www.crypnote.io/",
+  "supply": 10240000,
+  "email": "crypnote@crypnote.io",
+  "links": {
+    "github": "https://github.com/CrypNoteio",
+    "twitter": "https://twitter.com/CrypNoteio",
+    "discord": "https://discord.gg/CrypNote",
+	"NFT MINT":"https://nft.crypnote.io/",
```

Comment:
remove............. stop opening new prs

Reply:
Why? Then why am I in viewblock IO displays the relevant information of token

Reply:
Talking about this specific useless line